File: samples/snippets/csharp/VS_Snippets_CLR_System/system.Char.IsLetterOrDigit/CS/isletterordigit.cs
// <snippet6>
using System;

public class IsLetterOrDigitSample {
	public static void Main() {
		string str = "newline:\n";

		Console.WriteLine(char.IsLetterOrDigit('8'));		// Output: "True"
		Console.WriteLine(char.IsLetterOrDigit(str, 8));	// Output: "False", because it's a newline
	}
}
// </snippet6>

File: snippets/fsharp/VS_Snippets_CLR_System/system.Char.IsLetterOrDigit/FS/isletterordigit.fs
// <snippet6>
open System

let str = "newline:\n"

printfn $"{Char.IsLetterOrDigit '8'}"       // Output: "True"
printfn $"{Char.IsLetterOrDigit(str, 8)}"   // Output: "False", because it's a newline

// </snippet6>
